keepercommander/commands/enterprise.py

```diff
@@ -242,7 +242,7 @@ def __init__(self):
         self.team_keys = {}
 
     def execute_args(self, params, args, **kwargs):
-        if params.enterprise:
+        if params.enterprise or params.enterprise_id > 0:
```

Comment:
It looks like this change is not needed. If params.enterprise_id > 0 then params.enterprise should always be populated.

Reply:
This check was done before synching of the data was added to another method. Removed.